iot_config/src/org.rs

```diff
@@ -1,478 +1,167 @@
-}
-
-async fn add_constraint_update(
-    oui: u64,
-    net_id: NetIdField,
-    added_constraint: DevAddrConstraint,
-    db: &mut sqlx::Transaction<'_, sqlx::Postgres>,
-) -> Result<(), OrgStoreError> {
-    let helium_net_id: HeliumNetId = net_id
-        .try_into()
-        .map_err(|err: &'static str| OrgStoreError::InvalidUpdate(err.to_string()))?;
-    helium_netids::checkout_specified_devaddr_constraint(db, helium_net_id, &added_constraint)
-        .await
-        .map_err(|err| OrgStoreError::InvalidUpdate(format!("{err:?}")))?;
-    insert_helium_constraints(oui, net_id, &[added_constraint], db).await?;
-    Ok(())
-}
-
-async fn remove_constraint_update(
-    oui: u64,
-    net_id: NetIdField,
-    org_constraints: Option<&Vec<DevAddrConstraint>>,
-    removed_constraint: DevAddrConstraint,
-    db: &mut sqlx::Transaction<'_, sqlx::Postgres>,
-) -> Result<(), OrgStoreError> {
-    let helium_net_id: HeliumNetId = net_id
-        .try_into()
-        .map_err(|err: &'static str| OrgStoreError::InvalidUpdate(err.to_string()))?;
-    if let Some(org_constraints) = org_constraints {
-        if org_constraints.contains(&removed_constraint) && org_constraints.len() > 1 {
-            let remove_range = (u32::from(removed_constraint.start_addr)
-                ..=u32::from(removed_constraint.end_addr))
-                .collect::<Vec<u32>>();
-            db.release_addrs(helium_net_id, &remove_range).await?;
-            remove_helium_constraints(oui, &[removed_constraint], db).await?;
-            Ok(())
-        } else if org_constraints.len() == 1 {
-            return Err(OrgStoreError::InvalidUpdate(
-                "org must have at least one constraint range".to_string(),
-            ));
-        } else {
-            return Err(OrgStoreError::InvalidUpdate(
-                "cannot remove constraint leased by other org".to_string(),
-            ));
-        }
-    } else {
-        Err(OrgStoreError::InvalidUpdate(
-            "no org constraints defined".to_string(),
-        ))
-    }
-}
-
-async fn add_devaddr_slab(
-    oui: u64,
-    net_id: NetIdField,
-    addr_count: u64,
-    txn: &mut sqlx::Transaction<'_, sqlx::Postgres>,
-) -> Result<(), OrgStoreError> {
-    let helium_net_id: HeliumNetId = net_id
-        .try_into()
-        .map_err(|err: &'static str| OrgStoreError::InvalidUpdate(err.to_string()))?;
-    let constraints = helium_netids::checkout_devaddr_constraints(txn, addr_count, helium_net_id)
-        .await
-        .map_err(|err| OrgStoreError::SaveConstraints(format!("{err:?}")))?;
-    insert_helium_constraints(oui, net_id, &constraints, txn).await?;
-    Ok(())
-}
-
-async fn insert_helium_constraints(
-    oui: u64,
-    net_id: NetIdField,
-    devaddr_ranges: &[DevAddrConstraint],
-    db: impl sqlx::PgExecutor<'_>,
-) -> Result<(), sqlx::Error> {
-    let mut query_builder: sqlx::QueryBuilder<sqlx::Postgres> = sqlx::QueryBuilder::new(
+    let netid = sqlx::query_scalar::<_, i64>(
```

Comment:
should this be queried as a bigint now? the field for that table is defined in the new migration as an `integer` not a `bigint` and i don't see this anything changing the `netid.into()` to expect an i32 to cast the result into the `NetIdField` type unless I missed it